src/index.scss

```diff
@@ -227,7 +227,7 @@ header {
 
 .header-action-bar {
   background-color: #fff;
-  z-index: 1;
+  z-index: 2;
```

Comment:
why is this changed?

Reply:
The text editor action bar and header action bar had the same z-index value `1` which can create unpredictable UI overlap glitches for some browsers that why I increase this value according to UI stack.